.github/workflows/build-ci.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       run: |
         mkdir -p build &&
         pushd build &&
-        ../configure --enable-maintainer-mode --prefix=${HOME}/local --with-libctl=${HOME}/local/share/libctl ${MPICONF} &&
+        ../configure --enable-maintainer-mode --with-coverage --prefix=${HOME}/local --with-libctl=${HOME}/local/share/libctl ${MPICONF} &&
```

Comment:
The `--with-coverage` should actually have been added to this line:

https://github.com/NanoComp/meep/blob/ef2c9cad8a148c2602fa60e065a12352a89d6c21/.github/workflows/build-ci.yml#L152